packages/ui/src/assets/css/fileCard.css

```diff
@@ -0,0 +1,100 @@
+.p-card,
+.p-card-body,
+.p-card-content {
+  padding: 0 !important;
+}
```

Comment:
very risky css. This will overwrite css for the p-card component everywhere. Either use the local component's classname eg .file-card or update it so that this css only applies for this component only.